install.sh

```diff
@@ -3232,6 +3275,25 @@ execute_installation() {
         setup_shared_hook_repositories
         echo # For visual separation
     fi
+
+    finalize_installation || return 1
+}
+############################################################
+# Sets the install directory.
+#
+# Returns: None
+############################################################
+set_install_dir() {
```

Comment:
maybe `load_install_dir()` would be better, it would signal that this is setting the variable based on an existing setting.